openIMIS/openIMIS/settings.py

```diff
@@ -384,15 +384,14 @@ def SITE_URL():
 
 # Celery message broker configuration for RabbitMQ. One can also use Redis on AWS SQS
 CELERY_BROKER_URL = os.environ.get("CELERY_BROKER_URL", "amqp://rabitmq")
-if 'CELERY_RESULT_BACKEND' in os.environ:
-    CELERY_RESULT_BACKEND = os.environ.get("CELERY_RESULT_BACKEND")
+CELERY_RESULT_BACKEND = os.environ.get("CELERY_BROKER_URL", "amqp://rabitmq")
```

Comment:
```suggestion
if 'CELERY_RESULT_BACKEND' in os.environ:
    CELERY_RESULT_BACKEND = os.environ.get("CELERY_RESULT_BACKEND")
```

app crashes with CELERY_RESULT_BACKEND=amqp://rabitmq